package.json

```diff
@@ -49,15 +49,14 @@
     "rimraf": "^2.6.2"
   },
   "dependencies": {
-    "async": "^2.6.1",
     "base32.js": "~0.1.0",
     "bignumber.js": "^8.0.2",
     "cids": "~0.5.7",
     "datastore-core": "~0.6.0",
     "datastore-fs": "~0.7.0",
```

Comment:
Could you please add a dependency on https://github.com/ipfs/js-datastore-fs/pull/22 ?